.packit.yaml

```diff
@@ -18,6 +18,20 @@ jobs:
     srpm_build_deps:
       - make
 
+  - job: tests
+    identifier: revdeps
```

Comment:
my bad, forgot this one. Maybe `cockpit-podman-revdeps` / `cockpit-revdeps` would make it more explicit.

Reply:
@martinpitt how about including the minimum set of tests that you're hoping to see in podman gating, and doing multiple runs on them? Meaning, each test run includes all of the minimum tests. As the team gets comfortable (assuming they do), we can include more.

Btw, I pinged packit folks to check for `invisible` tasks, meaning something which won't show up in the github UI for PRs, but would show up on packit's dashboard. They don't have anything like that yet and have asked me to file a feature request. Assuming that's doable, combined with `ping specific people on failure` feature request, is that something that would work for you, if the team as a whole is not comfy with including these?

Reply:
> minimum set of tests that you're hoping to see in podman gating

Sorry, this is impossible to answer for me, at least in the way you hope. Let me turn the question around: Which of podman's features do you consider okay to break for users, and which not? E. g. a bare minimum test run of "I can start a container with `sleep 1` would not have caught the regression of breaking checkpoints (which would have landed in the next release).

> doing multiple runs on them

Like, re-run them 50 times to see how many times they fail? That I can do. We have a whole statistics database of test runs on our own CI, but not on the TF runs. This would be an interesting experiment.

> invisible tasks

That seems to make tracing back regressions unnecessarily hard for everyone, and is defeats the whole point of this exercise. The goal of this is to make your and my team's life more efficient and less frustrating by flagging and preventing regressions *before* they land, not sweep them under the rug (that's what we have to day). 

> "ping specific people on failure" feature

That absolutely makes sense, yes (see description). I believe it's not a hard blocker, but it would certainly make it much easier for both of us to handle problems when they happen. I'm happy to help the packit folks with implementing this. If you want to wait until that is a thing, that is certainly acceptable to me.

FTR: Of course your team has the right to say "we don't want this" or "we want to stop this experiment after one or six weeks" --  I'm just asking to give it a try. (I've gone through this discussion at least three times in my life, and after a few weeks nobody ever wanted to go back :grin: )

Reply:
> > doing multiple runs on them

> Like, re-run them 50 times to see how many times they fail

Procedural warning: There is no "restart" button for passed tests, and even for failed tests I don't have the privilege to press that button anyway (as I'm not a project member). So I'll have to do that through force-pushing 50 times. That's going to create a lot of notification noise, but at least we'll have a record of all these statuses in GitHub -- if that's okay with you?

Reply:
Ah, nevermind -- I can do the "50 re-runs" on my fork, like in https://github.com/martinpitt/podman/pull/1 . That avoids noise here, and also unnecessary runs of the other CI jobs (which would just be a waste of electrons).